ignite/engine/engine.py

```diff
@@ -327,7 +327,7 @@ def remove_event_handler(self, handler: Callable, event_name: Any):
             event_name: The event the handler attached to.
 
         """
-        self._assert_non_filtered_event(event_name)
+        # self._assert_non_filtered_event(event_name)
```

Comment:
Same here: remove commented code and if not yet done, we need a teston checking the correct behaviour with filtered events